Changelog.md

```diff
@@ -1,6 +1,7 @@
 ### 0.8.29 (unreleased)
 
 Language Features:
+ * Introduce basic syntax for specifying contract storage base location.
```

Comment:
```suggestion
 * Introduce syntax for specifying contract storage base location.
```

Unless we plan to introduce more complex syntax later?

Reply:
I think removing `basic` is more suited. In the issue there was mentions of "initial syntax" and I guess that is why I wrote basic syntax. But that's a detail which doesn't matter here imo.